lib/plugins/slack/index.js

```diff
@@ -59,73 +60,76 @@ module.exports = {
         dataCollector.addSummary('index', data);
         break;
       }
-    }
-  },
-  final(options) {
-    const slackOptions = merge({}, defaultConfig, options.slack);
-    const slack = new Slack(slackOptions.hookUrl);
-    const type = slackOptions.type;
-    const pageErrors = [];
-    const dataCollector = this.dataCollector;
-    let logo = 'https://www.sitespeed.io/img/slack/sitespeed-logo-slack.png';
 
-    let channel = slackOptions.channel;
-    if (channel && !channel.startsWith('#')) {
-      channel = `#${channel}`;
-    }
+      case 's3.finished': {
```

Comment:
this need to be only s3 if s3 is configured, else we should send to html.finished.